.github/workflows/cypress_workflow.yml

```diff
@@ -141,36 +143,88 @@ jobs:
       - run: npx cypress cache list
       - run: npx cypress cache path
 
-      - name: Run tests
+      - name: Setup spec files for Dashboards tests
+        if: ${{ inputs.specs == '' && matrix.group > 9 }}
+        shell: bash
+        run: |
+          IFS="," read -a SPEC_ARRAY <<< $(yarn --silent osd:ciGroup${{ matrix.group }})
+          DASHBOARDS_SPEC=''
+          for i in "${SPEC_ARRAY[@]}"; do
+          DASHBOARDS_SPEC+="cypress/integration/core_opensearch_dashboards/${i},"
+          done
+          echo "DASHBOARDS_SPEC=${DASHBOARDS_SPEC}" >> $GITHUB_ENV
+
+      - name: Run FT repo tests
+        if: ${{ matrix.group < 10 }}
```

Comment:
Do we plan to migrate all cypress test from FT repo into dashboard repo in the future?

Reply:
yes I think that will be ideal as we want feature and its tests to reside in the same place.